benchmarks/src/copy.cpp

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <algorithm>
+#include <benchmark/benchmark.h>
+#include <stddef.h>
+#include <string.h>
+#include <vector>
+
+#include <udt.hpp>
+
+template <typename Contained>
+static void handwritten_loop(benchmark::State& state) {

```

Comment:
Instead of marking each of these functions as `static` (presumably so that multiple source files can be linked into a single benchmark executable in the future), would it be better to put them into an unnamed namespace to avoid the repetition and potential for forgetting any occurrences?